src/components/NFTCover.vue

```diff
@@ -83,24 +98,33 @@ export default {
     imgPropsForPlaceholder() {
       return {
         class: [
-          'object-contain w-full',
+          'object-cover w-full',
           {
             'animate-pulse': !this.isLoaded,
+            'h-full min-h-[290px] max-w-[200px]': this.isNftBook,
           },
         ],
         loading: 'lazy',
         ...this.$attrs,
       };
     },
     resizedSrc() {
-      return getLikeCoResizedImageUrl(this.src, this.size);
+      return getImageResizeAPI(this.src, { width: this.size });
```

Comment:
`getLikeCoResizedImageUrl` only works for image served by like.co, while `getImageResizeAPI` was for old likerland og
Maybe we should merge both into one function

Reply:
For writing nfts they might, since uri override exists

Reply:
So the criteria for `getLikeCoResizedImageUrl` is URL includes `like.co`?

Reply:
Yes we can do that for now